src/SUMMARY.md

```
@@ -0,0 +1,26 @@
+# Summary
+
+[Inicio Rápido 🚀](README.md)
+
+- [Preguntas Frecuentes](./faq.md)
+	- [¿Qué es Mostro?](./what-is-mostro.md)
```

Markdownlint (0.37.0)

Expected: 2; Actual: 1
Unordered list indentation

(MD007, ul-indent)